packages/core/src/package.ts

```diff
@@ -72,11 +72,7 @@ const build = async (config: EnhancedConfig, packageName: string) => {
       entryFileNames(chunkInfo) {
         const entry = entries.find(
           ({ entryPath }) => chunkInfo.facadeModuleId === entryPath,
-        );
-
-        if (!entry) {
-          throw new Error('Unable to name entry file');
-        }
+        )!;
```

Comment:
I'd prefer the explicit error even if it should never happen. `!` is a spooky escape hatch that I've come to dislike because it's hard to see.

Reply:
It's impossible for an entry to not be found because each chunk [corresponds to an entry](https://github.com/seek-oss/crackle/blob/4ea357faf06affe880741581689508a3b3b348e6/packages/core/src/package/bundle.ts#L36).

More info here: https://rollupjs.org/guide/en/#outputentryfilenames

Reply:
I'd agree on keeping the explicit error. One for the increased readability (`!` *is* easy to miss), but two because if those two should diverge for whatever reason, it'll be a lot easier for us to work out what's happening.

Reply:
Done.